### PR TITLE
fixed css of the button of Sidebar + updated docker-compose.yml

### LINKED ISSUE #1056: Block To Text - CSS change
https://projects.raspberrypi.org/en/projects/editor-anime-expressions/editor

Image

### PR DESCRIPTION
**Pull request by KristinaDudnyk**
[#1056](https://github.com/RaspberryPiFoundation/editor-ui/issues/1056)

Screenshot 2024-08-06 at 15 26 47

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,10 +10,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
+## [0.25.5] - 2024-08-06
```
Comment (conorriches) [inline CHANGELOG.md:13]: We aren't releasing the version in this PR, so it can remain under "unreleased" :) 

```suggestion
```

**Review by conorriches — COMMENTED** [review]
nice! small tweak on the changelog but otherwise great!

File: CHANGELOG.md (hunk 1 above)
Comment (KristinaDudnyk) [reply CHANGELOG.md]: oki

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,10 +10,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Fixed
+
+- CSS fix for the bottom of Sidebar (Unreleased)
+
 ### Fixed
 
 - Small fix to ensure webpack is using the correct webSocketURL for live reloading\
 - Only register `editor-wc` once (#1052)
+  `
```
Comment (conorriches) [inline CHANGELOG.md:23]: I think this has snuck in :) 
```suggestion
  
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,10 +10,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
+## Unreleased
+
+### Fixed
+
+- CSS fix for the bottom of Sidebar (Unreleased)
```
Comment (conorriches) [inline CHANGELOG.md:17]: We can remove the unreleased bit, as it's under the Unreleased heading :) 
```suggestion
- CSS fix for the bottom of Sidebar
```

**Review by conorriches — COMMENTED** [review]
almost there! :)

**Review by conorriches — APPROVED** [review]
great!

**Pull request merged**